xml/System.ComponentModel.DataAnnotations/EnumDataTypeAttribute.xml

```diff
@@ -41,22 +41,24 @@
  The following example shows how to replace the numeric value of an enumeration with the corresponding declaration.  
   
 ```csharp  
-public enum eReorderLvl {  
-        zero = 0,  
-        five = 5,  
-        ten=10,  
-        fifteen=15,  
-        twenty=20,  
-        twenty_five=25,  
-        thirty=30  
-    }  
-    [MetadataType(typeof(Product_MD))]  
-    public partial class Product {  
-        public class Product_MD {  
-            [EnumDataType(typeof(eReorderLvl))]  
-            public object ReorderLevel { get; set; }  
-        }  
+public enum ReorderLevel
+{
+    Zero = 0,
+    Five = 5,
+    Ten = 10,
+    Fifteen = 15,
+    Twenty = 20,
+    TwentyFive = 25,
+    Thirty = 30
+}
+
+[MetadataType(typeof(ProductMD))]  
+public partial class Product {  
```

Comment:
can you also make this and the next line to use Allman style so they're all consistent?

Reply:
Oh yes, good spotted! 👍